src/map-converter.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @flow
+ */
+
+export function convertToMap<V>(mapObject: {[string]: V}): Map<string, V> {
```

Comment:
I tried to generify this function so that can take any `K`, such that it would be possible to generate `MapOfRoleDefinitions`, but sadly Flow started erroring about the key not being a string. As far as I know, `ARIARoleDefintionKey` is a string, but Flow didn't want to allow this assignment. Sadly I am not a user of Flow, so maybe I am missing some solution here?